### ISSUE TITLE
Filters do not seem to work with boolean values or 0

### ISSUE THREAD

**Issue by DeividasK** [prescottprue/react-redux-firebase#25]
Hello,

It seems that ordering by boolean values does not work.

```javascript
@firebaseConnect([
  '/goals#orderByChild=done&equalTo=true'
])
```

It does not work with `0` as well

```javascript
@firebaseConnect([
  '/goals#orderByChild=done&equalTo=0'
])
```

It works fine with other values. Any ideas why this might be the case?

**Comment by prescottprue** [comment]
Do you have the values stored as booleans on Firebase? Or as strings?

Going to attempt to replicate with the simple example.

**Comment by prescottprue** [comment]
I was able to replicate and find where the bug was coming from.

The fix is included in [`v1.1.4`](https://github.com/prescottprue/react-redux-firebase/releases/tag/v1.1.4).
